components/authentication-framework/org.wso2.carbon.identity.application.authentication.endpoint/pom.xml

```diff
@@ -42,6 +42,12 @@
         </dependency>
         <dependency>
             <groupId>javax.servlet</groupId>
+            <artifactId>jsp-api</artifactId>
+            <version>2.0</version>
```

Comment:
Define dependency version in the parent pom and inherit those, if not it's a maintenance nightmare.